plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2834,20 +2874,27 @@ private void testStatsBasedRepartitionData(boolean ctas)
     }
 
     private void testRepartitionData(Session session, String sourceRelation, boolean ctas, String partitioning, int expectedFiles)
+    {
+        testRepartitionData(session, sourceRelation, Optional.empty(), -1, ctas, partitioning, expectedFiles);
+    }
+
+    protected void testRepartitionData(Session session, String sourceRelation, Optional<String> sourceOrderBy, int limitSourceRows, boolean ctas, String partitioning, int expectedFiles)
```

Comment:
nit: You could use `Optional.empty()` instead of `-1` for `limitSourceRows. `-1` feels like a magic string.

Reply:
Ok may be a little more clear.